.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = E501
```

Comment:
do you know if other services projects are ignoring this rule universally?  i think there's also a way to do this for individual lines if that were the case.